### ISSUE TITLE
prettify_on_save doesn't work at all in 2.1.16

### ISSUE THREAD

**Issue by Pastafarianist** [SublimeHaskell/SublimeHaskell#388]
Sublime Text 3156, fully updated Manjaro Linux. SublimeHaskell config:

```
{
    "use_improved_syntax": true,

    "backends": {
        "No backend": {
            "default": true,
            "backend": "none"
        }
    },

    "prettify_on_save": true,
    "stylish_options": [
        "--config",
        "/path/to/config1/.stylish-haskell.yaml",
        // "/path/to/config2/.stylish-haskell.yaml",
    ],

    "add_word_completions": true,
    "add_default_completions": true,
    "auto_completion_popup": true,
}
```

<kbd>Ctrl</kbd>+<kbd>S</kbd> does not trigger prettifier, while <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> does. Applies to files both with and without a corresponding project.

**Comment by bscottm** [comment]
@Pastafarianist: Show me. How do I reproduce? I can't seem to at all. It works for me, with and without files in `cabal` projects.

**Comment by bscottm** [comment]
@Pastafarianist: Your config, Build 3156, file not associated with a `cabal`:

<img width="1365" alt="screen shot 2018-01-10 at 8 35 51 am" src="https://user-images.githubusercontent.com/355395/34783875-8fe20dce-f5e1-11e7-8771-c65548ad12c0.png">

**Comment by Pastafarianist** [comment]
I've done the following:

1. Created a new directory `~/temp` and opened it in Sublime.
2. Created a new file `test.hs` with the following contents:

```haskell
import Control.Monad (forM_)
import Control.Concurrent.Async (wait, async)

main :: IO()
main = do
    a1 <- async $ putStrLn "hello"
    a2 <- async $ putStrLn "world"
    forM_ [a1, a2] wait
```

3. Brought up the console.
4. Modified `indenters.py` by adding the following line at the beginning of `do_prettify`:

```python
print("=== IN DO_PRETTIFY ===")
```

This is what I got after all that:

![image](https://user-images.githubusercontent.com/1315874/34801529-3ec1da48-f67a-11e7-8213-71c4fca09c8b.png)

Then I switched to the tab with `test.hs` and pressed <kbd>Ctrl</kbd>+<kbd>S</kbd>. The console immediately disappeared. After bringing it back I saw the traceback I mentioned in https://github.com/SublimeHaskell/SublimeHaskell/issues/389.

![image](https://user-images.githubusercontent.com/1315874/34801560-69f11bca-f67a-11e7-9acb-29f9c33bebe1.png)

**Comment by bscottm** [comment]
@Pastafarianist: That's exactly the bug fixed in 2.1.16. The MD5 checksums for `check_lint.py` and `indenters.py` should be:

    c9e4e0d30a444578987085d11cc7084b  check_lint.py
    3a4418c10249e0d35dd9e07ee073b6cf  indenters.py

**Comment by Pastafarianist** [comment]
I apologize: I guess I haven't restarted Sublime after the latest update.
I confirm that (during the two tests I've just performed) the bug does not manifest both with files in a project and standalone ones.
